pkg/k8shandler/fluentd.go

```diff
@@ -400,6 +436,12 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, trustedCABundleCM *v1.Co
 		{Name: metricsVolumeName, MountPath: "/etc/fluent/metrics"},
 	}
 
+	exporterContainer.VolumeMounts = []v1.VolumeMount{
+		{Name: logVolumeMountName, MountPath: logVolumePath},
+	}
+	// default parameters --> exporterContainer.Command = []string{"/usr/local/bin/log-file-metric-exporter -verbosity=2 -dir=/var/log/containers -http=:2112"}
+	exporterContainer.Command = []string{"/bin/sh", "-c", "/usr/local/bin/log-file-metric-exporter  -verbosity=2  -dir=/var/log/containers -http=:2112"}
```

Comment:
it is working fine and helped me to debug from container terminal.

Reply:
It is an extra process that _might_ cause subtle changes in signal handling behavior now that there are 2 processes instead of one. I can't prove that it _will_, but to quote Murphy's Law: if anything can go wrong, it will. I'm not paranoid, they really are out to get me.

Reply:
I say this because we have seen in the past that apparently simple changes that start multiple processes (e.g. letting fluentd fork workers) seem to have caused pods to hang on exit.

Reply:
+1 We should be able to run this:
* by defaulting to the image entrypoint
* calling the command directly without needing to wrapper in bash

Reply:
@jcantrill sorry i am not clear on this. Are you saying cmd should execute directly this binary as the below
exporterContainer.Command = []string{"/usr/local/bin/log-file-metric-exporter  -verbosity=2  -dir=/var/log/containers -http=:2112"}

or the entrypoint should be set as []string{"/usr/local/bin/log-file-metric-exporter  -verbosity=2  -dir=/var/log/containers -http=:2112"}